src/utils/__tests__/sorter.test.js

```diff
@@ -295,11 +295,43 @@ describe('util/sorter', () => {
                 },
             ],
         };
+        const annotations = {
```

Comment:
Do we want to move these mocks to a separate file so they can be reused? Looks like the same data is in a couple places now.

Reply:
Good idea